lib/git/lib.rb

```diff
@@ -1040,10 +1040,19 @@ def commit_tree(tree, opts = {})
       t.write(opts[:message])
       t.close
 
+      # Adapted from activesupport Array.wrap
+      parents = if opts[:parents].nil?
```

Comment:
I read up on `Array.wrap` but don't understand why you would choose that implementation instead of `Kernel#Array`.  I am interested to hear why you would go this route instead of reducing this change to:

```Ruby
arr_opts += Array(opts[:parent]).map { |p| ['-p', p] }.flatten
```

I think calling `#to_a` if `#to_ary` doesn't exist would be the desired behavior.